src/DataValues/QuantityValue.php

```diff
@@ -237,128 +202,6 @@ public function getAmount() {
-	 * data point. For example, the uncertainty interval may be defined to be that part of a
-	 * normal distribution that is required to cover the 95th percentile.
-	 *
-	 * @since 0.1
-	 *
-	 * @return float
-	 */
-	public function getUncertainty() {
-		return $this->upperBound->getValueFloat() - $this->lowerBound->getValueFloat();
-	}
-
-	/**
-	 * Returns a DecimalValue representing the symmetrical offset to be applied
-	 * to the raw amount for a rough representation of the uncertainty interval,
-	 * as in "amount +/- offset".
-	 *
-	 * The offset is calculated as max( amount - lowerBound, upperBound - amount ).
-	 *
-	 * @since 0.1
-	 *
-	 * @return DecimalValue
-	 */
-	public function getUncertaintyMargin() {
-		$math = new DecimalMath();
-
-		$lowerMargin = $math->sum( $this->amount, $this->lowerBound->computeComplement() );
-		$upperMargin = $math->sum( $this->upperBound, $this->amount->computeComplement() );
-
-		$margin = $math->max( $lowerMargin, $upperMargin );
-		return $margin;
-	}
-
-	/**
-	 * Returns the order of magnitude of the uncertainty as the exponent of
-	 * last significant digit in the amount-string. The value returned by this
-	 * is suitable for use with @see DecimalMath::roundToExponent().
-	 *
-	 * @example: if two digits after the decimal point are significant, this
-	 * returns -2.
-	 *
-	 * @example: if the last two digits before the decimal point are insignificant,
-	 * this returns 2.
-	 *
-	 * Note that this calculation assumes a symmetric uncertainty interval,
-	 * and can be misleading.
-	 *
-	 * @since 0.1
-	 *
-	 * @return int
-	 */
-	public function getOrderOfUncertainty() {
-		// the desired precision is given by the distance between the amount and
-		// whatever is closer, the upper or lower bound.
-		//TODO: use DecimalMath to avoid floating point errors!
-		$amount = $this->amount->getValueFloat();
-		$upperBound = $this->upperBound->getValueFloat();
-		$lowerBound = $this->lowerBound->getValueFloat();
-		$precision = min( $amount - $lowerBound, $upperBound - $amount );
-
-		if ( $precision === 0.0 ) {
-			// If there is no uncertainty, the order of uncertainty is a bit more than what we have digits for.
-			return -strlen( $this->amount->getFractionalPart() );
-		}
-
-		// e.g. +/- 200 -> 2; +/- 0.02 -> -2
-		// note: we really want floor( log10( $precision ) ), but have to account for
-		// small errors made in the floating point operations above.
-		// @todo: use bcmath (via DecimalMath) to avoid this if possible
-		$orderOfUncertainty = floor( log10( $precision + 0.0000000005 ) );
-
-		return (int)$orderOfUncertainty;
-	}
-
-	/**
-	 * Returns the number of significant figures in the amount-string,
-	 * counting the decimal point, but not counting the leading sign.
-	 *
-	 * Note that this calculation assumes a symmetric uncertainty interval, and can be misleading
-	 *
-	 * @since 0.1
-	 *
-	 * @return int
-	 */
-	public function getSignificantFigures() {
```

Comment:
I'm happy to discuss factoring the significant digits stuff out, but let's please do that in a later pull request, so we keep steps small and don't bock one change because of discussion about the other.
I agree that it should target the same major release, though.